From 018e32a21b8fb04c07440ef2b8d389f3805aefb5 Mon Sep 17 00:00:00 2001
From: chewitt <christianshewitt@gmail.com>
Date: Thu, 28 May 2020 09:07:11 +0000
Subject: [PATCH 33/33] HACK: prevent stdout/stderr on videoconsole

Severl devices now have CONFIG_DM_VIDEO enabled which causes stdout/stderr
to appear on videoconsole (HDMI). Remove videoconsole so that early u-boot
boot remains silent unless you are using the UART/serial console.
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 851cba807f..52ba6eab02 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -29,7 +29,7 @@
 #define CONFIG_BMP_32BPP
 #define CONFIG_SPLASH_SCREEN
 #define CONFIG_SPLASH_SCREEN_ALIGN
-#define STDOUT_CFG "vidconsole,serial"
+#define STDOUT_CFG "serial"
 #else
 #define STDOUT_CFG "serial"
 #endif
-- 
2.17.1

